templates/base.html.twig

```diff
@@ -553,6 +545,12 @@
             <div id="search-engine"></div>
         </div>
 
+        <link rel="stylesheet" type="text/css" href="{{ asset('css/cookieconsent.min.css') }}" />
```

Comment:
this stylesheet was declared after `app.css`

Reply:
✅